lib/fastlane/plugin/toolkit/source/configmanager.rb

```diff
@@ -58,27 +67,41 @@ def for_lane(lane_name)
 			end
 		end
 
-		# Override Configfile configuration for a specific platform.
+		# Override Configfile parameters for a specific platform.
 		#
 		# platform_name  - Symbol representing a platform name.
 		# block - Block to execute to override configuration values.
 		#
 		def for_platform(platform_name)
-			options = [Global.platform, ENV.fetch("FASTLANE_PLATFORM_NAME")].map(&:to_s)
+			options = [Toolkit.platform, ENV.fetch("FASTLANE_PLATFORM_NAME", nil)].map(&:to_s)
```

Comment:
Just curious, why is this `Toolkit.platform` instead of `Platform.platform`? Isn't the addition at the top of this class a typealias?

Reply:
ruby is case sensitive. so `platform` and `Platform` are different things. The top level `Platform` module houses constants - like `Platform::IOS` or `Platform::ANDROID`. `Toolkit` is a module with a class property of `platform` that is the currently selected platform.

the reason class Config needs to know about the global module `Platform` is bc app code Configfiles may contain references to constants in that module like `Platform::IOS`

```
module Platform
   IOS = :ios
   ANDROID = :android
end

module Toolkit
   def platform
      @platform
   end
end
```